src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -99,6 +100,14 @@ def create_storage_account(cmd, resource_group_name, account_name, sku=None, loc
             queue_encryption_service = EncryptionService(enabled=True, key_type=encryption_key_type_for_queue)
             params.encryption.services.queue = queue_encryption_service
 
+    if any([routing_choice, publish_microsoft_endpoints, publish_internet_endpoints]):
+        RoutingPreference = cmd.get_models('RoutingPreference')
+        params.routing_preference = RoutingPreference(
+            routing_choice=routing_choice,
+            publish_microsoft_endpoints=publish_microsoft_endpoints,
+            publish_internet_endpoints=publish_microsoft_endpoints
```

Comment:
`publish_internet_endpoints=publish_microsoft_endpoints` Why is this logic? What is the function of `publish_Internet_endpoints`?

Reply:
Goo catch!